pkg/apis/autoscaling/annotations.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package autoscaling
+
+// OtherMetricsAnnotation is the annotation which holds non-CPU-utilization HPA metric
+// specs when converting the `Metrics` field from autoscaling/v2alpha1
+const OtherMetricsAnnotation = "autoscaling.alpha.kubernetes.io/metrics"
```

Comment:
The current names OtherMetricsAnnotation and CurrentMetricsAnnotation are similar and easy to confuse.
I would change the names so that it is explicit which one of them is status and which one of the is spec, e.g.: `OtherMetricsAnnotation`  -> `MetricsSpecAnnotation` and `CurrentMetricsAnnotation` -> `MetricsStatusAnnotation`.